docs/custom-image.md

```diff
@@ -0,0 +1,115 @@
+In this section we will learn how we can create a custom BitOps image with plugins and additional utilities. 
+
+
+
+### Clone Bitops source
+First thing is first, if you haven't already clone the github bitops repo locally
+`git clone https://github.com/bitovi/bitops.git`
+
+<hr/>
+
+
+### Modify the *bitops.config.yaml*
+Using your preferred editor, open the root level `bitops.config.yaml`
+
+This file is used to configure the BitOps image.
+
+[Example bitops.config.yaml](https://github.com/bitovi/bitops/tree/plugins/prebuilt-config)
+
+<hr/>
+
+### bitops.config.yaml
+
+<br/>
+
+#### BitOps "official" image
+Below is an example of how the "official" image of bitops is configured. 
+
+As you can see there are two sections we need to be aware of; Plugins and Deployments. 
+
+<br/>
+
+#### plugins
+Defines a plugin and the source for that plugin. 
+
+<br/>
+
+#### Deployment
+Defines alias', the sequence of executions for those alias' and the alias' related plugin.
+
+
+### official image
```

Comment:
move this section above the `#### plugins` and `#### Deployment` headings (under the "as you can see ..." sentence)

Reply:
I looked into this and I'm not sure I agree with the placement. 

Can we talk more about this if you really think it's valuable?

Reply:
the intent behind my suggestion is to follow the pattern of
- heading
- short description
- example config
- describe the specific things in the example config

Happy to chat